client/blindai/client.py

```diff
@@ -531,8 +531,8 @@ def upload_model(
 
         Args:
             model (str): Path to Onnx model file.
-            tensor_inputs (Union[List[Any], List[List]]): A list describing multiple inputs of the model.
-            tensor_outputs (Union[ModelDatumType, List[ModelDatumType]): A list describing multiple inputs of the model. Defaults to {"index_0": ModelDatumType.F32}.
+            tensor_inputs (Union[Tuple[List[int], ModelDatumType], List[Tuple[List[int], ModelDatumType]]): The list of input fact and datum types for each input grouped together in lists, describing the different inputs of the model.
```

Comment:
Is that really necessary? This writing in the Union makes it very complex and confusing.

Reply:
Yeah, you are right. It even confuses me as well.
Okay, then I guess I will take it out, and make it only 
```python
tensor_inputs(List[Tuple[List[int], ModelDatumType]])
tensor_outputs(List[ModelDatumType])
```